compose/internal/service/access_control.go

```diff
@@ -170,6 +170,17 @@ func (svc accessControl) CanRunAutomationTrigger(ctx context.Context, r *automat
 	return svc.can(ctx, types.AutomationScriptPermissionResource.AppendID(r.ID), "run", permissions.Allowed)
 }
 
+// @todo actual controll
+func (svc accessControl) CanReadImportSession(ctx context.Context) bool {
```

Comment:
No need for AC for the record importer.
If the user can create records (on module) she can import them.
Session access control is done solely on ownership.